src/server/rest/v1/service/SiteAreaService.ts

```diff
@@ -364,7 +365,10 @@ export default class SiteAreaService {
     }
     siteArea.smartCharging = filteredRequest.smartCharging;
     siteArea.accessControl = filteredRequest.accessControl;
-    siteArea.siteID = filteredRequest.siteID;
+    if (siteArea.siteID !== filteredRequest.siteID) {
+      await ChargingStationStorage.updateChargingStationsSite(req.tenant, filteredRequest.id, filteredRequest.siteID);
```

Comment:
Always call this method.
To put after the save.

Reply:
Done